cmd/main.go

```diff
@@ -759,6 +759,25 @@ func getOverlay(ctx *Context, system *config.System, module string) (string, err
 	return "", nil
 }
 
+func getExampleOverlay(ctx *Context, system *config.System, module string) (string, error) {
+
+	repo, _, err := config.FindRepository(ctx.Repos, module)
+	if err != nil {
+		return "", err
+	}
+
+	for _, repoOverlay := range repo.Overlays {
+		for _, systemOverlay := range system.Overlays {
+			if repoOverlay == systemOverlay && strings.HasPrefix(systemOverlay, "example") {
```

Comment:
Maybe the prefix it checks for should be "examples-" ? Functionally it doesn't matter, but it makes it more obvious to me what it's doing.